tests/test_standard_model.py

```diff
@@ -371,107 +375,3 @@ def test_gml_import_export(tmpdir_factory, model):
         assert node
         assert (ConceptNode.to_dict(node['object']) ==
                 ConceptNode.to_dict(orig_node['object']))
-
```

Comment:
These tests having to do with unique key stuff have been moved into `test_transform_stage.py` since standard model no longer does unique key gen.